src/api/User/types.ts

```diff
@@ -125,3 +125,7 @@ export interface IMergeUsersParams {
   from: number;
   to: number;
 }
+
+export interface ICountSharedUserAssetsParams {
+  id: number;
```

Comment:
why id and not count?
also, should we have some additional data for the most shared user? with the name of the user that has received most assets + count of those assets?